pkg/database/db.go

```diff
@@ -1,15 +1,40 @@
 package database
 
 import (
-	"database/sql"
-	// "github.com/go-sql-driver/mysql"
+	"errors"
+	"fmt"
+	"github.com/jinzhu/gorm"
+	"log"
+	"os"
 )
 
-// Init initializes an instance of the database
-func Init() (*sql.DB, error) {
-	db, err := sql.Open("mysql", "root@cloudsql(project-id:instance-name)/dbname")
+// MustInit initializes an instance of the database
+func MustInit() *gorm.DB {
+	log.Println("Initializing Cloud SQL connection...")
+
+	// Extract database credentials from environment
+	username := os.Getenv("SQL_USERNAME")
+	password := os.Getenv("SQL_PASSWORD")
+	connectionName := os.Getenv("SQL_CONNECTION_NAME")
+	databaseName := os.Getenv("SQL_DATABASE")
+
+	if username == "" {
+		panic(errors.New("SQL_USERNAME var is empty"))
+	}
+	if password == "" {
+		panic(errors.New("SQL_PASSWORD var is empty"))
+	}
+	if connectionName == "" {
+		panic(errors.New("SQL_CONNECTION_NAME var is empty"))
+	}
+	if databaseName == "" {
+		panic(errors.New("SQL_DATABASE var is empty"))
+	}
+	dataSourceName := fmt.Sprintf("%s:%s@cloudsql(%s)/%s?charset=utf8&parseTime=True&loc=Local", username, password, connectionName, databaseName)
+
+	db, err := gorm.Open("mysql", dataSourceName)
```

Comment:
Does this mean that the development db is also Cloud SQL? Should we not use a local DB?

Reply:
Yes, but can create a dev db in the cloud instance. The instance houses several databases. We would just need to change an environment variable. `main` would be prod, and `dev` could be used for development.